WalletWasabi/WabiSabi/Models/DecompositionAlgs/Coin.cs

```diff
@@ -0,0 +1,40 @@
+using NBitcoin;
+using System;
+
+namespace WalletWasabi.WabiSabi.Models.DecompositionAlgs
+{
+	public record Coin : IComparable
+	{
+		public Money Amount { get; init; } = Money.Zero;
+		public const int OutputVbytes = 31;
+		public const decimal InputVbytes = 68.5m;
+
+		private int HashCode { get; } = new Guid().GetHashCode();
+
+		public Money EffectiveValue(FeeRate feeRate)
+		{
+			long sats = (long)Math.Floor(feeRate.SatoshiPerByte * InputVbytes);
```

Comment:
This could be done with: `var fees = feeRate.GetFee(InputVBytes);`.

Reply:
InputVbytes is decimal. GetFee can take int only.

Reply:
This was removed. The theoretical question, what if InputVbytes is not a whole number like it was before?

Reply:
I was mistaken when I made it a decimal in the notebook, I didn't realize vbytes always implicitly rounds up `(int)((3+WU)/4)`, the way both core and nbitcoin handle it is with ints and we should stick to that